### DISCUSSION TITLE
Does CONUS404-BA account for time zone differences between CONUS404 and Daymet before doing bias correction?

### DISCUSSION THREAD

**Discussion by jds485** [hytest-org/hytest#513] [Dataset Questions]
I am wondering if the bias correction accounted for different time zones for the correction dataset (Daymet) and simulation data (CONUS404).

**Comment by amsnyder** [comment]
Thanks for the question Jared. I haven't worked with Daymet data much, but just from poking around online a bit, it sounds like Daymet uses a 24-hour day based on local time ([source](https://planetarycomputer.microsoft.com/dataset/daymet-daily-na)), while CONUS404 times are UTC times.

I am also seeing is this ([source](https://daymet.ornl.gov/overview)):

> The Daymet calendar is based on a standard calendar year. All Daymet years have 1 - 365 days, including leap years. For leap years, the Daymet database includes leap day. Values for December 31 are discarded from leap years to maintain a 365-day year.

CONUS404 is not constrained to a 365 day year, so this is another difference.

Hopefully @aubreyd can tag someone from NCAR who can help us answer this.

**Reply by aubreyd** [reply]
Tagging @YongxinFred

**Reply by YongxinFred** [reply]
Thank you Jared for the question. Thank you Amelia for the response. We did take into account the time zone differences between CONUS404 (which uses UTC time) and Daymet (which uses Local time) when we developed the bias corrections.

**Reply by jds485** [reply]
Okay, great! Thanks. I think that should be documented in the ScienceBase data release metadata.

**Reply by YongxinFred** [reply]
Jared, that is a good suggestion. Thanks. Yes, let's add that in the ScienceBase data release metadata. Yongxin

**Reply by amsnyder** [reply]
Thanks @jds485 - I will add it to our task queue to do a data release revision to add this information. Thanks @YongxinFred for helping answer this question!

**Reply by jds485** [reply]
> Thank you Jared for the question. Thank you Amelia for the response. We did take into account the time zone differences between CONUS404 (which uses UTC time) and Daymet (which uses Local time) when we developed the bias corrections.

Following up on this, does that mean that the CONUS404-BA data are in local time? Or are they also in UTC time?

**Reply by aubreyd** [reply]
They are also in UTC

**Comment by jds485** [comment]
One more related question (feel free to move to another discussion if you think it's better). Does the CONUS404 Pseudo Global Warming scenario contain bias adjustments? If not, how would we apply bias adjustments to temperature and precipitation for the PGW dataset?

**Reply by rviger** [reply]
USGS/NCAR plan to make a Data Release of bias-adjusted temperature and precipitation values from CONUS404-PGW using the same method and weightings as were used for the CONUS404-historical (https://www.sciencebase.gov/catalog/item/64f77acad34ed30c20544c18).

**Reply by YongxinFred** [reply]
Thank you Roland for the reply!
